conans/client/command.py

```diff
@@ -590,6 +592,8 @@ def config(self, *args):
         set_subparser.add_argument("item", help="'item=value' to set")
         init_subparser.add_argument('-f', '--force', default=False, action='store_true',
                                     help='Overwrite existing Conan configuration files')
+        list_subparser.add_argument('-r', '--raw', default=False, action='store_true',
```

Comment:
Avoid the --raw for now, lets keep it simple. Users shouldn't parse this output for anything.

Reply:
Done